CameraXBasic/app/src/main/res/layout/fragment_camera.xml

```diff
@@ -17,11 +17,10 @@
 <androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/camera_container"
-    android:background="@android:color/black"
```

Comment:
Why are you removing this line? I think black background it makes the rotation animation (for devices that don't support seamless) and app startup look better.

Reply:
Reverted that; Sorry, I've undone one line too much.